src/PerfView/RangeUtils.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Globalization;
+
+namespace PerfView
+{
+    /// <summary>
+    /// Provides utility operations for <see cref="double"/> ranges.
+    /// </summary>
+    internal static class RangeUtils
+    {
+        /// <summary>
+        /// Array of list separators built from the current culture.
+        /// </summary>
+        /// <remarks>The user can set any character as a separator and that character can turn a regular expression invalid.</remarks>
+        private static readonly string[] listSeparators = new string[] { CultureInfo.CurrentCulture.TextInfo.ListSeparator };
```

Comment:
Is there a reason that you need to allocate a new array for this?  You should just be able to use the string `ListSeparator` as input to `string.Split` in `TryParse`.

Reply:
That overload only exists for .NET Core. It doesn't exist for .NET Framework.

Reply:
Yes, you're right.